pkg/cmd/server/api/v1/types_test.go

```diff
@@ -352,12 +380,21 @@ func TestMasterConfig(t *testing.T) {
 			Extensions: []internal.AssetExtensionsConfig{{}},
 		},
 		DNSConfig: &internal.DNSConfig{},
+		AdmissionConfig: internal.AdmissionConfig{
+			PluginConfig: map[string]internal.AdmissionPluginConfig{ // test config as an embedded object
+				"plugin": {
+					Configuration: runtime.EmbeddedObject{Object: &AdmissionPluginTestConfig{}},
+				},
+			},
+			PluginOrderOverride: []string{"plugin"}, // explicitly setting a value because the field is omitempty
+		},
 	}
 	serializedConfig, err := writeYAML(config)
 	if err != nil {
 		t.Fatal(err)
 	}
 	if string(serializedConfig) != expectedSerializedMasterConfig {
+		t.Errorf("\n\n%s\n\n%s\n\n", string(serializedConfig), string(expectedSerializedMasterConfig))
```

Comment:
sorry didn't mean to leave that in. The string wasn't matching and it was driving me crazy because they looked identically. It turns out that at some point I had inserted tabs instead of spaces in the expected string.